### ISSUE TITLE
bicyle=yes rendering

### ISSUE THREAD

**Issue by road-runner-osm** [cyclosm/cyclosm-cartocss-style#672]
I would also like to hear other perspectives, but I think bicycle=yes is not well rendered.

Bicycle=yes is (after bicycle=designated) a pretty strong indicator that a road/track is also made for cycling. For example, the wiki suggests using it for these paths or signs: example https://wiki.openstreetmap.org/wiki/File:Australia_road_sign_R8-1.svg https://wiki.openstreetmap.org/wiki/File:Z239ZZ1022_10GehwegRadfahrerFrei1.jpg So bicycle=yes indicates that a road infrastructure is maybe not made foremost and in law for bikes, but it is made _also_ for it.

Nevertheless it is not rendered differently than the standard path without bicycle=yes in Cyclosm. But paths without bicycle=yes are in many countries not allowed or are often not suitable for bikes (because they are mountain paths, to steep, too bumpy...). 
Thunderforest OpenCycleMap does render bicycle=yes like bicycle=designated by the way. So maybe Cyclosm could also render it? Maybe also in blue or some lighter blue?

**Comment by waterced** [comment]
I agree with @[Phyks](https://github.com/Phyks). To me, these signs:
> https://wiki.openstreetmap.org/wiki/File:Australia_road_sign_R8-1.svg https://wiki.openstreetmap.org/wiki/File:Z239ZZ1022_10GehwegRadfahrerFrei1.jpg

means the way should be tagged with ` bicycle=designated`

**Comment by Atrate** [comment]
I would like to express support for this idea. In Poland, there's quite a few footways that are formally sidewalks but legally usable by bikes due to specific conditions (next to a high-speed road) and it would be great to see such sidewalks rendered differently (as otherwise you can't cycle on them). We tag them with `bicycle=yes`. OpenCycleMap already does that.
